packages/odie-client/src/index.tsx

```diff
@@ -23,7 +23,10 @@ export const OdieAssistant: React.FC = () => {
 	return (
 		<div className="chatbox">
 			<div className="chat-box-message-container" id="odie-messages-container">
-				<MessagesContainer currentUser={ currentUser } />
+				<MessagesContainer
+					currentUser={ currentUser }
+					newInteractionsBotSlug={ newInteractionsBotSlug }
```

Comment:
`MessagesContainer` should get this value from the context. No need to pass it down.